src/Raven.Server/Config/Categories/StorageConfiguration.cs

```diff
@@ -78,6 +78,12 @@ public class StorageConfiguration : ConfigurationCategory
         [LegacyConfigurationEntry("Raven/TransactionsModeDuration")]
         public int TransactionsModeDuration { get; set; }
 
+        [Description("Maximum concurrent flushes")]
+        [DefaultValue(10)]
+        [ConfigurationEntry("Raven/Storage/MaxConcurrentFlushes")]
+        [LegacyConfigurationEntry("Raven/MaxConcurrentFlushes")]
```

Comment:
@aviviadi LegacyConfigurationEntry should only be used when you are migrating an option from 3.x so it would be easier for us to create a mapping between 3.x and 4.0 options for the clients. Is this the case?